content/docs/command-reference/remote/modify.md

```diff
@@ -616,8 +621,8 @@ more information.
   [delegated] to the service account if needed.
 
   ```dvc
-  $ dvc remote modify myremote \
-                 gdrive_service_account_user_email 'myemail-addr'
+  $ dvc remote modify --local myremote \
+                      gdrive_service_account_user_email 'myemail-addr'
```

Comment:
...and here.

Reply:
I would not make this one sensitive as well

Reply:
Reverted.